packages/util/src/asyncEventEmitter.ts

```diff
@@ -0,0 +1,194 @@
+import { eachSeries } from 'async'
```

Comment:
Just so that we do not forget: please add license and a source reference here, both to the async-eventemitter package as well as the corresponding definitely-typed TS files.